data/docs/migration/migrate-from-elk/traces.mdx

```diff
@@ -0,0 +1,56 @@
+---
+date: 2025-05-06
+id: migrate-from-elk-traces
+title: Migrate traces from ELK Stack
+description: Migrating traces from ELK Stack to SigNoz. How to migrate traces from ELK to SigNoz.
+---
+## **Migrating Traces: From Elastic APM to SigNoz**
+
+Migrating distributed tracing involves replacing the proprietary [Elastic APM agents](https://www.elastic.co/docs/solutions/observability/apm/elastic-apm-agents) within your applications with standard [OpenTelemetry SDKs](https://opentelemetry.io/docs/languages/) and configuring them to send trace data to SigNoz.50
+
+**Replacing Elastic APM Agents:**
+
+* **Core Task:** The fundamental step is to remove the Elastic APM agent dependencies and initialization code from your applications and replace them with the equivalent OpenTelemetry SDKs and instrumentation [libraries for the specific proframming language](/docs/instrumentation/).
+* **Instrumentation Approach:**  
+  * **Automatic Instrumentation:** OpenTelemetry offers automatic instrumentation capabilities for many popular frameworks and libraries, similar to Elastic APM's approach.13 Examples include the OTel Java agent, @opentelemetry/auto-instrumentations-node for Node.js, and various opentelemetry-instrumentation-\* packages for Python.52 This often provides the quickest path to getting traces with minimal code changes.  
```

Comment:
Why are there strange numbers? 13, 52, 47?

Reply:
Had used an AI to reformat some content, it might have left it there, will proofread all the docs again to weed out such issues.